css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+h2 {
+  margin-left: 48%;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  color: rgb(237 96 91);
+  letter-spacing: 0.1rem;
+}
+
+label {
+  display: block;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.6rem;
+  padding-left: 33.5%;
+  padding-bottom: 2px;
+  color: black;
+  letter-spacing: 0.5px;
+}
+
+input {
+  display: block;
```

Comment:
You'll want to change the border style of your inputs too. The default is some weird 3D effect, but we just want a solid border.